<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC-BSD Update Center</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/update-center.png</normaloff>:/update-center.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="res.qrc">:/images/beta.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>This is early beta not full featured software for testing only!</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>&amp;More details...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="JailIndicator" name="jailIndicatorWidget" native="true"/>
    </item>
    <item>
     <widget class="RebootWidget" name="RebootW" native="true"/>
    </item>
    <item>
     <widget class="QTabWidget" name="mainTab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Main">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QStackedWidget" name="mainStatesStack">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_5">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>82</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="res.qrc">:/images/sysupdater.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Your system is fully updated</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="checkAllButton">
                <property name="text">
                 <string>Check for &amp;Updates</string>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>49</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_6">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>System</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="IndicatorWidget" name="sysIndicator" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="packagesIndicatorGB">
              <property name="title">
               <string>Packages</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="IndicatorWidget" name="pkgIndicator" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="pkgIndictorLine">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Software</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="IndicatorWidget" name="pbiIndicator" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="updateAllButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Update all</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/images/install.png</normaloff>:/images/install.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/sysupdates.png</normaloff>:/images/sysupdates.png</iconset>
       </attribute>
       <attribute name="title">
        <string>System update details</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="SysDetailsWidget" name="sysDetailsW" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/pkgupdates.png</normaloff>:/images/pkgupdates.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Package update details</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="PkgDetailsWidget" name="pkgDetailsW" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/pbiupdates.png</normaloff>:/images/pbiupdates.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Software update details</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="PBIDetailsWidget" name="pbiDetailsW" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLogs">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionLast_system_update_log"/>
    <addaction name="actionLast_package_update_log"/>
    <addaction name="actionLast_software_update_log"/>
    <addaction name="separator"/>
    <addaction name="actionBase_system_update_log"/>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>&amp;Configure</string>
    </property>
    <addaction name="actionJail"/>
    <addaction name="actionSystem_branches"/>
    <addaction name="actionUpdate_set"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuLogs"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLast_system_update_log">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Last &amp;system update log</string>
   </property>
  </action>
  <action name="actionLast_package_update_log">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Last &amp;package update log</string>
   </property>
  </action>
  <action name="actionLast_software_update_log">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Last soft&amp;ware update log</string>
   </property>
  </action>
  <action name="actionUpdate_set">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Update set</string>
   </property>
  </action>
  <action name="actionSystem_branches">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;System branches</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/window-close.png</normaloff>:/images/window-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionJail">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Remote Jail</string>
   </property>
  </action>
  <action name="actionBase_system_update_log">
   <property name="text">
    <string>&amp;Base system update history</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IndicatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">indicatorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SysDetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">sysdetailswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PkgDetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">pkgdetailswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PBIDetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">pbidetailswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RebootWidget</class>
   <extends>QWidget</extends>
   <header location="global">rebootwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JailIndicator</class>
   <extends>QWidget</extends>
   <header location="global">jailindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
